src/db/redux/ServerConnections/ServerConnection.ts

```diff
@@ -12,10 +12,12 @@ model.address = 'localhost';
 
 export interface ServerConnectionState {
   serverConnection: ServerConnection;
+  valid: boolean;
 }
 
 const initialState: ServerConnectionState = {
   serverConnection: new PgClient(model), // TODO maybe change this later
+  valid: true, // TODO change this to false later
```

Comment:
Do we want to make this value dependent on if we're in dev or prod?
```suggestion
  valid: process.env.NODE_ENV === 'production' ? false : true;
```

Reply:
Added the change